DbMocker.Tests/DbMockTableTests.cs

```diff
@@ -15,13 +15,9 @@ public void Mock_ReturnsScalar_MockTable_Test()
         {
             var conn = new MockDbConnection();
 
-            MockTable table = MockTable.Empty()
-                                 .AddColumns("Col1")
-                                 .AddRow(11);
+            MockTable table = MockTable.Empty().AddColumns("Col1").AddRow(11);
```

Comment:
Can you avoid to reformat the existing code? This generates a PR that becomes very difficult to compare.

Reply:
sorry, CSharpier kicking in. Will fix that